sponsorship_compassion/model/contracts.py

```diff
@@ -15,7 +15,7 @@
 from openerp.tools import DEFAULT_SERVER_DATE_FORMAT as DF
 
 from datetime import datetime
-from dateutil.relativedelta import relativedelta
+from dateutil.relativedelta import relativedelta

```

Comment:
I corrected the settings in my editor. These kinds will probably still show up a few times though...